### ISSUE TITLE
Whenever i load a map into chunky the 2d map doesnt load at all just shows black,

### ISSUE THREAD

**Issue by shortfire715** [llbit/chunky#500]
I'm running minecraft vs 1.13.2

**Comment by Codewow** [comment]
This is occurring on mine as well. Utilizing MC 1.13.2, and the Chunky Snapshot1 for 1.4.6

The issue appears to be the program failing to load any of the textures.

Failed to load textures:
	shulkerLime
	concrete_powder_orange
	rose_bush_top
	pumpkin_face_on
	glass_pane_top_pink
	shulkerMagenta
	cauldron_side
	crafting_table_front
	shulkerOrange
	repeater_off
	... and 434 more

**Comment by mrglitchyr** [comment]
This problem happened to me too. It looks like the current chunky version is incompatible with minecraft 1.13.2. So I decided to open the previous world from 1.12 and It somewhat worked. But some textures are missing. The light gray wool texture were replaced with white wool. And the planks and slabs were not from vanilla. I also wanted to know the release date of the Chunky version is compatible with the latest release MC version.
